### PR TITLE
Create update.json

### PR DESCRIPTION
**Pull request by delmicio**
I'm not sure if this is the correct way for this project. 
May be could be grabbed from https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/chosen

### CODE REVIEW

**Comment by delmicio** [comment]
@jimaek could you help me a bit here? please

**Comment by megawac** [comment]
You need to specify `files` in the update.json

**Comment by delmicio** [comment]
@megawac  `files` is optional.
I've never worked with npm, I don't understand how to find the files there.
I wonder how can cdnjs bot have it updated there.

@pfiller, @kenearley, @stof, @koenpunt, @tjschuck.

**Comment by megawac** [comment]
files is not optional. `basePath` and `include`/exclude are optional. Anyway your issue now is that files is a string and not an object.

As for the files to include do an npm install for chosen and list the files you want included on the cdn

**Comment by delmicio** [comment]
@megawac  Thanks you! I've done that and found the files. Although no minified version is provided this way :-1:

File: files/chosen/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "chosen",
```
Comment (megawac) [inline files/chosen/update.json]: I think you want this name to be chosen-js, `chosen` on npm looks like the wrong thing

**Pull request merged**